save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserPermissions.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.info
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property isPermittedCreateContest permission for create contests
+ * @property isPermittedToBulkUpload permission for upload COSV files
+ * @property organizationName name of organization
+ */
+@Serializable
+data class UserPermissions(
+    val isPermittedCreateContest: Boolean = false,
+    val isPermittedToBulkUpload: Boolean = false,
+    val organizationName: String? = null,
```

Comment:
remove this field and move it to flags (make it a complex)

Reply:
Fixed